world_x: 3
world_y: 2
world_border: 0.022
beac_radius: 0.048
beac_l: 0.1
beac_border: 0.022